sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/PartialMethodNoImplementation.cs

```diff
@@ -74,29 +76,32 @@ private static void CheckForCandidatePartialDeclaration(SyntaxNodeAnalysisContex
         private static void CheckForCandidatePartialInvocation(SyntaxNodeAnalysisContext context)
         {
             var invocation = (InvocationExpressionSyntax)context.Node;
+
+            if (!(invocation.Parent is StatementSyntax statement))

```

Comment:
Move the syntax check before checking for symbol so that we do as much as possible on the non-costy side.